fluidreview/views_test.py

```diff
@@ -0,0 +1,37 @@
+"""Tests for FluidReview views"""
+import pytest
+from django.core.urlresolvers import reverse
+from rest_framework import status
+
+from fluidreview.models import WebhookRequest
+
+
+@pytest.mark.parametrize("data", ['', "some data", '{"json": "body"}'])
+def test_webhook(db, data, client, settings):  # pylint: disable=unused-argument
```

Comment:
Could add `pytestmark = pytest.mark.django_db` before the tests instead of using `db` as a parameter for each test.  But either way works.